### PR TITLE
Create add-to-project workflow

### LINKED ISSUE #51: Add action to triage HTAN-data-curator new issues into backlog
As already used for data-models repo

### PR DESCRIPTION
**Pull request by adamjtaylor**
Adds all issues to the data-models project. Closes #51

### CODE REVIEW

File: .github/workflows/add-to-project.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.3.0
+        with:
+          # You can target a repository in a different organization
+          # to the issue
+          project-url: https://github.com/orgs/ncihtan/projects/4
+          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```
Comment (elv-sb) [inline .github/workflows/add-to-project.yml:18]: ADD_TO_PROJECT_PAT or PATH? @adamjtaylor

Comment (adamjtaylor) [reply .github/workflows/add-to-project.yml:18]: `ADD_TO_PROJECT_PAT` is correct - it is secret token that the action needs to have permission to write in the project.

But this review reminds me that I actually need to create it in this repo (as this was copied from the data-models repo

Comment (elv-sb) [reply .github/workflows/add-to-project.yml:18]: Ok, other than that, this looks good to go

**Review by elv-sb — APPROVED** [review]
Looks good to go

**Pull request merged**